docs/digger/admin/installing-jenkins-plugins.asciidoc

```diff
@@ -1,21 +1,22 @@
 == Installing Jenkins plugins
-Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be installed by including them in Ansible playbook or using the Jenkins UI.
-
 NOTE: Plugins for Jenkins can be found at https://plugins.jenkins.io
 
+Digger will by default install required Jenkins plugins. If more Jenkins plugins are required, they can be installed by listing them in a task in Ansible playbook or using Jenkins UI.
+
 === Using Ansible Playbook
-The `jenkins_plugins` parameter can be overridden to include new plugins. The
-structure of a Jenkins plugin is as follows.
+https://github.com/aerogear/aerogear-digger-installer/blob/master/configure-buildfarm/defaults/main.yml[Digger Installer Ansible playbook^]. 
+The `jenkins_plugins` play can be modified to include new plugins. 
 
+.The structure of a sample Ansible task:
 [source,yaml]
 ----
 -
   name: sample-plugin
   version: 1.20
-  archive: http://updates.jenkins-ci.org/download/plugins/sample-plugin/1.20/sample-plugin.hpi
+  archive: http://updates.jenkins.io/download/plugins/sample-plugin/1.20/sample-plugin.hpi
```

Comment:
jenkins-ci.org is correct.

Reply:
both work for me, should i bother to change it? it seems that jenkins.io will be a new url

Reply:
resolved